tools/cldr-apps/js/src/esm/cldrText.js

```diff
@@ -412,7 +412,7 @@ const strings = {
   E_NOT_STARTED:
     "The SurveyTool is still starting up. Please wait a bit and hit Reload.",
   E_SPECIAL_SECTION:
-    "An error occurred while trying to ${what}, the server said that those items aren't visible in the Survey Tool.\nPerhaps the URL is incorrect or an item was deprected?",
+    "An error occurred while trying to ${what}.\nThe XPath “${surveyCurrentId}” in locale “${surveyCurrentLocale}” is not visible in the SurveyTool.\nPerhaps the URL is incorrect or an item was deprecated?",
```

Comment:
A question: "is not visible" in the Survey Tool sounds like "it exists but the ST can't display it". Is that what you meant?

Reply:
yes